mysqld_exporter.go

```diff
@@ -252,7 +252,13 @@ func main() {
 		http.Handle("/", landingPage)
 	}
 	http.HandleFunc("/probe", handleProbe(enabledScrapers, logger))
-
+	http.HandleFunc("/-/reload", func(w http.ResponseWriter, r *http.Request) {
+		if error := c.ReloadConfig(*configMycnf, *mysqldAddress, *mysqldUser, *tlsInsecureSkipVerify, logger); error != nil {
+			level.Info(logger).Log("msg", "Error reloading host config", "file", *configMycnf, "err", error)
```

Comment:
`error` is a Go type, so it can be ambiguous. We generally use `err`. This should also be at least a Warning level error.

```suggestion
		if err = c.ReloadConfig(*configMycnf, *mysqldAddress, *mysqldUser, *tlsInsecureSkipVerify, logger); err != nil {
			level.Warn(logger).Log("msg", "Error reloading host config", "file", *configMycnf, "error", err)
```